recipes/grpcio/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "grpcio" %}
+{% set version = "1.6.3" %}
+{% set sha256 = "e9662782c58bc21be26163b78136eaed091dfd45fac699cb711ee9eeeb7e2f9b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
```

Comment:
Add `toolchain` as a `build` requirement.

Reply:
@david-gang, please see the comment above

Reply:
@isuruf I fixed the issues and still have errors in circle ci. Thanks for the help